### PR TITLE
Dev 4310 Set up xlwings for sample LUSID Excel sheets

### PR DESCRIPTION
**Pull request by thomasdotred**
Tests that LUSID Excel - Setting up your IBOR template Global Equity Fund.xlsx, view scopes actually displays correct scopes

### CODE REVIEW

File: tests/tutorials/test_excel_tutorial.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+def get_date(date, sht):
+    date_at = sht.range(date).value
+    if date_at:
+        date_at = datetime(date_at.year, date_at.month, date_at.day, tzinfo=pytz.utc)
+    return date_at
+
+
+class Scopes(unittest.TestCase):
+    # This test validates that the data displayed in "LUSID Excel -Setting up your IBOR template Global Wquity
```
Comment (rizwansaeed) [inline tests/tutorials/test_excel_tutorial.py]: typo 'Wquity'

File: tests/tutorials/test_excel_tutorial.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        sht = wb.sheets(sheet_name)
+        self.assertTrue(sht, msg="Error loading sheet: " + sheet_name)
+
+        scopes = sht.range(range_containing_scopes).value
+        self.assertTrue(scopes, msg="Error loading scopes from range: " + range_containing_scopes)
+
+        excel_scopes = []
+        scopes = scopes[0:9]
+        for scope in scopes:
+            if scope:
```
Comment (rizwansaeed) [inline tests/tutorials/test_excel_tutorial.py]: can `scope` ever be `None`? If it is I think you can replace this with something like

`excel_scopes = [scope for scope in scope[0:9] if scope is not None]`

File: tests/tutorials/test_excel_tutorial.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+                excel_scopes.append(scope)
+
+        # get scopes to validate against
+        response = self.scopes_api.list_scopes()
+        validation_scopes = []
```
Comment (rizwansaeed) [inline tests/tutorials/test_excel_tutorial.py]: as above, use python list comprehension

File: tests/tutorials/test_excel_tutorial.py (hunk 2 above)
Comment (thomasdotred) [reply tests/tutorials/test_excel_tutorial.py]: Equity changed

File: tests/tutorials/test_excel_tutorial.py [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+        for scope_i in response.values[:9]:
+            validation_scopes.append(scope_i.scope)
+
+        # Assert that the scopes from Excel == scopes from LUSID
+        self.assertEqual(excel_scopes, validation_scopes, msg="Scopes from " + book_name + "do not equal validation "
+                                                                                           "set")
+
+    def test_view_portfolios(self):
```
Comment (rizwansaeed) [inline tests/tutorials/test_excel_tutorial.py]: we should probably split out each workbook into a separate test case

File: tests/tutorials/test_excel_tutorial.py [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+        self.assertTrue(scope, msg="Error loading scope from location: " + scope_location)
+
+        portfolios_validation = []
+        portfolios_response_validation = self.portfolios_api.list_portfolios_for_scope(
+            scope=scope,
+            effective_at=effective_at if effective_at else "",
+            as_at=as_at if as_at else ""
+        )
+
+        for data in portfolios_response_validation.values:
```
Comment (rizwansaeed) [inline tests/tutorials/test_excel_tutorial.py]: use list comprehension

File: tests/tutorials/test_excel_tutorial.py (hunk 4 above)
Comment (thomasdotred) [reply tests/tutorials/test_excel_tutorial.py]: These are both the same workbook, just different sheets. Each workbook has about 10 sheets

**Pull request merged**